server/models/migrations/next/migrate.sql

```diff
@@ -252,4 +252,12 @@ CREATE TABLE IF NOT EXISTS `required_inventory_scan` (
 ) ENGINE=InnoDB DEFAULT CHARACTER SET = utf8mb4 DEFAULT COLLATE = utf8mb4_unicode_ci;
 
 INSERT IGNORE INTO unit VALUES
-  (313, 'Inventory Scans Management', 'TREE.REQUIRED_INVENTORY_SCANS', 'Inventory Scans Management', 307, '/required/inventory/scans');
\ No newline at end of file
+  (313, 'Inventory Scans Management', 'TREE.REQUIRED_INVENTORY_SCANS', 'Inventory Scans Management', 307, '/required/inventory/scans');
+
+/**
+Issue: 6501
+@author: jniles
+@date: 2022-04-06
+*/
+DELETE FROM role WHERE unit_id = 167;
```

Comment:
Shouldn't this be DELETE FROM permission???

Reply:
Oh, that is probably true.  Sorry.